lib/bundler/definition.rb

```diff
@@ -256,9 +256,39 @@ def lock(file)
         "#{File.expand_path(file)}"
     end
 
+    # Returns the version of Bundler that is creating or has created
+    # Gemfile.lock. Used in #to_lock.
+    def lock_version
+      lockfile = @lockfile_contents
+      lock_ver = ""
+      curr_ver = ""
+
+      if !lockfile.empty? and lockfile.match(/^  \[(.*)\]$/)
```

Comment:
Since the idiomatic `!thing.empty?` in ruby is `thing.any?`, I think it would probably make sense to switch to that.